athletics.lic

```diff
@@ -35,6 +39,106 @@ class Athletics
   def done_training?
     DRSkill.getxp('Athletics') >= @end_exp
   end
+  
+  def stop_play
+    bput('stop play', 'You stop playing your song', 'In the name of', "But you're not performing")
+    Flags['ct-song'] = true
```

Comment:
Not right now, but we should revisit this if `athletics` is integrated into `crossing-training`.